addons/mail/security/mail_security.xml

```diff
@@ -351,4 +351,167 @@
             <field name="perm_create" eval="True"/>
             <field name="perm_unlink" eval="False"/>
         </record>
+
+        <record id="ir_rule_discuss_poll_group_system" model="ir.rule">
+            <field name="name">discuss.poll: admin full access</field>
+            <field name="model_id" ref="mail.model_discuss_poll"/>
+            <field name="groups" eval="[Command.link(ref('base.group_system'))]"/>
+            <field name="domain_force">[(1, '=', 1)]</field>
+        </record>
+        <record id="ir_rule_discuss_poll_read_all" model="ir.rule">
+            <field name="name">discuss.poll: anyone can read polls of accessible channels</field>
+            <field name="model_id" ref="mail.model_discuss_poll"/>
+            <field name="groups"
+                eval="[
+                    Command.link(ref('base.group_user')),
+                    Command.link(ref('base.group_portal')),
+                    Command.link(ref('base.group_public')),
+                ]"
+            />
+            <field name="domain_force">
+                [
+                    "|",
+                        "&amp;",
+                            ("channel_id.channel_type", "!=", "channel"),
```

Comment:
These rules will miss livechat override, among other things.

Having ACL depending on a relation field (that can be edited, most likely) is never a good idea either.

Can't we make the model fully technical and use sudo?